core/src/main/java/io/confluent/rest/ProxyConnectionFactory.java

```diff
@@ -0,0 +1,821 @@
+/*
+ * Copyright 2025 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.rest;
```

Comment:
@silvanli-confluent This was a workaround in 7.9.x which is based on Jetty-9. We wouldn't need this in 8.0.x and above as it is based on Jetty 12. It might be better to backport  https://github.com/confluentinc/rest-utils/pull/584 . CC: @manhdatcao